docker-app/qfieldcloud/core/serializers.py

```diff
@@ -535,3 +546,44 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField(source="team_organization")
+
+    class Meta:
+        model = Team
+        fields = ("username", "organization")
+
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.CharField()
+
+    def to_internal_value(self, data: Dict[str, Any]) -> Dict[str, Any]:
+        validated_data = super().to_internal_value(data)
+        email_or_username = data.get("member")
+
+        try:
+            existing_user = User.objects.fast_search(email_or_username)
+
+            if TeamMember.objects.filter(member=existing_user):
+                raise serializers.ValidationError(
+                    {
+                        "member": f"A member with username '{email_or_username}' already exists."
+                    }
+                )
+
+            validated_data["member"] = existing_user
+
+        except User.DoesNotExist:
+            raise serializers.ValidationError(
+                {
+                    "member": f"A user with username '{email_or_username}' does not exist."
```

Comment:
```suggestion
                    "member": _('User "{}" does not exist.').format(email_or_username),
```

Reply:
Issue here with _() as mypy does not seems to recognize it. Solved with `from django.utils.translation import gettext as _
`